ci/testnet-sanity.sh

```diff
@@ -52,14 +52,4 @@ echo "--- $NET_URL: validator sanity"
   timeout 10s ./multinode-demo/validator.sh "$NET_URL" 2>&1 | tee log
```

Comment:
Don't we want to check `log` for a panic?

Reply:
I'm getting a bit confused, I've apparently broken CI because this script runs somewhere I don't know how to reproduce locally